### ISSUE TITLE
Feature requests

### ISSUE THREAD

**Issue by Ni55aN** [retejs/rete#632]
Please check out [the discussion](https://github.com/retejs/rete/discussions/631)

**Comment by delafer** [comment]
Віталій Столяров я бачу ти справжній патріот! Я впевнений у тебе цілком призовний вік між 18 до 60. Я сподіваюся ти записався до збройних лав України? відколи github став платформою для всіляких політичних агітацій, гасел та іншого?
є куди простіший шлях допомогти своїй Батьківщині, скажи, в якому районі Харкова живеш, і я скажу, де найближчий військкомат.

Vitaliy Stolyarov, I can see you are a real patriot! I'm sure you are of a fully conscripted age between 18 and 60. I hope you've enlisted in the armed forces of Ukraine. since when did github become a platform for all kinds of political campaigning, slogans and other things?
There is a much easier way to help your homeland. Tell me what district of Kharkiv you live in, and I'll tell you where the nearest military enlistment office is.

**Comment by Ni55aN** [comment]
@delafer thanks for bringing up this issue and showing concern for the defense of Ukraine.

Regarding your question: GitHub is a platform for sharing the results of intellectual work with the community freely, “as is”, where authors have the right to add any content as long as it doesn't violate platform rules. What you're referring to as political slogans are responses to Russia's full-scale invasion.

I appreciate your willingness to help, so besides the advice, you can do a bit more — donate to the Ukrainian army or charitable funds. You can find them at the link https://stand-with-ukraine.pp.ua (works for Germany too).

**Comment by delafer** [comment]
> Regarding your question: GitHub is a platform for sharing the results of intellectual work with the community freely, “as is”, where authors have the right to add any content as long as it doesn't violate platform rules. What you're referring to as political slogans are responses to Russia's full-scale invasion.
> 
> I appreciate your willingness to help, so besides the advice, you can do a bit more — donate to the Ukrainian army or charitable funds. You can find them at the link

I don't send money to war criminals and Nazis. And besides, you've already gotten enough

**Comment by delafer** [comment]
@Ni55aN 
GitHub’s Acceptable Use Policies are clear on the type of content and activities that are not allowed on the platform. Specifically, GitHub does not allow content or activity that is unlawful, promotes unlawful activities, or threatens or incites violence toward any individual or group, especially on the basis of who they are.

Advertising and campaigning in support of one country in a war with another could potentially fall under these categories, especially if it involves unlawful activities or incitement of violence. Such content could also be considered as harassment or abuse toward individuals or groups, which is against GitHub’s policies.

Furthermore, GitHub’s guidelines emphasize the importance of being respectful and not posting content that is discriminatory or abusive toward any individual or group. This would include content that could be seen as an insult to the citizens of a country that such campaigning is against.

In conclusion, while GitHub supports free expression and collaboration, its policies do not permit content that could be harmful or offensive to others, including political advertising and campaigning related to war efforts. It’s important to review and adhere to GitHub’s Acceptable Use Policies to ensure a positive and lawful experience for all users.

**Comment by Ni55aN** [comment]
@delafer 
> Sorry, I don't send money to any war criminals and Nazis

What country are you talking about? Oh... nevermind. 

> don't write that it's Russian propaganda

That's it! It looks sick, so I'll skip to P.S.

> I'm a Ukrainian and I was born and studied in Kiew

Can anyone at **Dr. Güldener Gruppe** proove it, **Alexander Bock**?

> I addressed you in Ukrainian language, but you preferred to answer in English

It's easy - the community is mostly English-speaking, so it makes sense that members understand replies to messages like yours

> could be harmful or offensive to others

Is it harmful or offensive to you when someone supports a country that has been invaded?

**Comment by Ni55aN** [comment]
@delafer 
> Vitaly, I could easily disprove all your theses

You can't even answer simple questions (they are not rhetorical). "Yes" or "no", along with a brief explanation. Instead, you write extensive text closely aligned with the theses of Russian propaganda:
![image](https://github.com/retejs/rete/assets/8259641/10ec64e3-f82c-4a05-9953-d22b6c45812c)

Given the lack of progress in your ability to succinctly express your thoughts in the appropriate ticket, you are now subject to a 30-day block. Please take this time as an opportunity to reflect on your answers to existing questions and to concisely state your position.

I'm open to discussions like this. I don't believe that here, on GitHub, we should refrain from discussing issues not directly related to development. Remarkably, you've raised political topics here, which only proves that Open Source cannot be apolitical.

Btw, we can leave these comments (perhaps they will be useful in the future), please do not delete them.

**Comment by lgrkvst** [comment]
This... is the feature requests thread...?!
